nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -25,6 +25,8 @@
 
 - [Playerctld](https://github.com/altdesktop/playerctl), a daemon to track media player activity. Available as [services.playerctld](option.html#opt-services.playerctld).
 
+- [Oathkeeper](https://www.ory.sh/oathkeeper/), authenticate, authorize and mutate any incoming network traffic, using a Zero Trust / BeyondCorp security model
```

Comment:
That's a package not a service. We usually don't add new packages to release notes, otherwise it will be too long

Reply:
Thanks for the review, I removed the package from the release notes.